pkg/azureclients/armclient/azure_armclient.go

```diff
@@ -496,7 +496,7 @@ func (c *Client) PutResourcesInBatches(ctx context.Context, resources map[string
 	return responses
 }
 
-// PutResourceWithDecorators puts a resource by resource ID
+// putResourceWithDecorators puts a resource by resource ID and waits for response
```

Comment:
Why `PutResourceWithDecorators` in lowercase?

Reply:
oh my bad, will fix it.